src/main/java/org/sert2521/gamename/Drivetrain/Drivetrain.kt

```diff
@@ -0,0 +1,51 @@
+package org.sert2521.gamename.Drivetrain
+
+import edu.wpi.first.wpilibj.command.Command
+import edu.wpi.first.wpilibj.command.Subsystem
+import org.sert2521.gamename.FRONT_LEFT_MOTOR
+import org.sert2521.gamename.FRONT_RIGHT_MOTOR
+import org.sert2521.gamename.OI
+import org.sert2521.gamename.REAR_LEFT_MOTOR
+import org.sert2521.gamename.REAR_RIGHT_MOTOR
+import org.strongback.components.Motor
+import org.strongback.drive.TankDrive
+import org.strongback.hardware.Hardware
+
+/**
+ * This is the Drivetrain subsystem, which is in charge of controlling the speeds of the individual
+ * talons and converting joystick input to movement speeds.
+ */
+object Drivetrain : Subsystem() {
```

Comment:
Well but this isn't our real code. Since we're getting rid of the logger, we'll only have like 3 files or something. And I meant the root root directory as in `..gamename`.

As for organization, not quite. The Drivetrain will probably stay in the root directory and we'll have stuff like `driving`, `shooting`, `gears` and whatnot. For commands like you're suggesting that could be used literally anywhere, I would just put them in a root `utils` package since that what they really are but with instances.